src/en/charms.md

```diff
@@ -248,23 +274,27 @@ with the `juju models` command. Your output should be similar to the
 following:
 
 ```bash
-Model       Owner  Status     Machines  Cores  Access  Last connection
-controller  admin  available         1      -  admin   just now
-default*    admin  available         2      -  admin   just now
+Controller: google-controller
+
+Model       Cloud/Region     Status     Machines  Cores  Access  Last
+connection
+controller  google/us-east1  available         1      4  admin   just now
+default     google/us-east1  available         1      1  admin   1 hour ago
+prod*       google/us-east1  available         2      2  admin   55 seconds ago
 ```
 
-To remove the default model, type `juju destroy-model default` and enter 'Y' to 
-accept the warning that this step will destroy all machines, applications,
-data and other resources associated with the default model. A few moments
-later, depending on the complexity of your model, you should find it no longer
-listed in the output of the `models` command. 
+To remove the 'prod' model we've been using for the MediaWiki deployment, type
+`juju destroy-model prod` and enter 'Y' to accept the warning that this step
+will destroy all machines, applications, data and other resources associated
+with that model. A few moments later you should find it no longer listed in the
+output of the `juju models` command. 
 
-If you want to start again with a clean default model, restoring Juju to the
-state it was in before we deployed the MediaWiki charm, you can create one 
-with:
+If you want to start again with a clean model, restoring Juju to the state it
+was in before we deployed the MediaWiki charm, you can re-create the 'prod'
+model with:
 
 ```bash
-juju add-model default
+juju add-model prod
```

Comment:
The point here was to return everything to as it was. We used the default model, deleted it then recreated it again.
Notwithstanding anything else, 'prod' isn't a great name for a model.